manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackActionTransformer.java

```diff
@@ -26,16 +26,8 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
 
         resolvedAction.setType(KafkaTopicAction.TYPE);
 
-        newParameters.put(KafkaTopicAction.TOPIC_PARAM, generateKafkaTopicName(processorId));
+        newParameters.put(KafkaTopicAction.TOPIC_PARAM, resourceNamesProvider.getProcessorTopicName(processorId));
 
         return resolvedAction;
     }
-
```

Comment:
+1 Thanks for removing the spurious comment

Reply:
you're welcome :wink: